configs/defconfig.esp32

```diff
@@ -0,0 +1,75 @@
+CONFIG_BOOTLOADER_LOG_LEVEL_NONE=y
+CONFIG_ESPTOOLPY_FLASHSIZE_4MB=y
+CONFIG_ARDUHAL_ESP_LOG=y
+CONFIG_COMPILER_OPTIMIZATION_SIZE=y
+CONFIG_COMPILER_CXX_EXCEPTIONS=y
+CONFIG_COMPILER_STACK_CHECK_MODE_NORM=y
+CONFIG_COMPILER_WARN_WRITE_STRINGS=y
+CONFIG_BT_ENABLED=y
+CONFIG_BTDM_CTRL_MODE_BTDM=y
+CONFIG_BTDM_SCAN_DUPL_CACHE_SIZE=20
+CONFIG_BT_BTC_TASK_STACK_SIZE=8192
+CONFIG_BT_CLASSIC_ENABLED=y
+CONFIG_BT_A2DP_ENABLE=y
+CONFIG_BT_SPP_ENABLED=y
+CONFIG_BT_HFP_ENABLE=y
+CONFIG_BT_STACK_NO_LOG=y
+CONFIG_BT_ALLOCATION_FROM_SPIRAM_FIRST=y
+CONFIG_BT_BLE_DYNAMIC_ENV_MEMORY=y
+CONFIG_ESP32_SPIRAM_SUPPORT=y
+CONFIG_SPIRAM_MALLOC_ALWAYSINTERNAL=4096
+CONFIG_SPIRAM_TRY_ALLOCATE_WIFI_LWIP=y
+CONFIG_SPIRAM_MALLOC_RESERVE_INTERNAL=0
+CONFIG_ESP32_ULP_COPROC_ENABLED=y
+CONFIG_ESP32_XTAL_FREQ_AUTO=y
+CONFIG_ETH_SPI_ETHERNET_DM9051=y
+CONFIG_ETH_SPI_ETHERNET_W5500=y
+CONFIG_HTTPD_MAX_REQ_HDR_LEN=1024
+CONFIG_HTTPD_WS_SUPPORT=y
+CONFIG_ESP_IPC_TASK_STACK_SIZE=1024
+CONFIG_ESP_SYSTEM_EVENT_TASK_STACK_SIZE=2048
+CONFIG_ESP_MAIN_TASK_STACK_SIZE=4096
+CONFIG_ESP_INT_WDT_TIMEOUT_MS=300
+CONFIG_ESP_TASK_WDT_PANIC=y
+# CONFIG_ESP_TASK_WDT_CHECK_IDLE_TASK_CPU1 is not set
+CONFIG_ESP_TIMER_TASK_STACK_SIZE=4096
+CONFIG_ESP32_WIFI_STATIC_RX_BUFFER_NUM=8
+CONFIG_ESP32_WIFI_STATIC_TX_BUFFER_NUM=8
+CONFIG_ESP32_WIFI_CACHE_TX_BUFFER_NUM=16
+# CONFIG_ESP32_WIFI_ENABLE_WPA3_SAE is not set
+CONFIG_FATFS_CODEPAGE_850=y
+CONFIG_FATFS_LFN_STACK=y
+# CONFIG_FMB_CONTROLLER_SLAVE_ID_SUPPORT is not set
+CONFIG_FMB_TIMER_PORT_ENABLED=y
+CONFIG_FREERTOS_HZ=1000
+# CONFIG_FREERTOS_ASSERT_ON_UNTESTED_FUNCTION is not set
+CONFIG_FREERTOS_WATCHPOINT_END_OF_STACK=y
+CONFIG_FREERTOS_IDLE_TASK_STACKSIZE=1024
+CONFIG_FREERTOS_FPU_IN_ISR=y
+CONFIG_HEAP_POISONING_LIGHT=y
+CONFIG_LOG_DEFAULT_LEVEL_ERROR=y
+# CONFIG_LOG_COLORS is not set
+CONFIG_LWIP_MAX_SOCKETS=16
+CONFIG_LWIP_ETHARP_TRUST_IP_MAC=y
+# CONFIG_LWIP_DHCP_DOES_ARP_CHECK is not set
+CONFIG_LWIP_DHCP_RESTORE_LAST_IP=y
+CONFIG_LWIP_DHCP_OPTIONS_LEN=128
+CONFIG_LWIP_TCP_SYNMAXRTX=6
+CONFIG_LWIP_TCP_MSS=1436
+CONFIG_LWIP_TCP_RTO_TIME=3000
+CONFIG_LWIP_TCPIP_TASK_STACK_SIZE=2560
+CONFIG_LWIP_TCPIP_TASK_AFFINITY_CPU0=y
+CONFIG_LWIP_SNTP_MAX_SERVERS=3
+CONFIG_LWIP_DHCP_GET_NTP_SRV=y
+# CONFIG_MBEDTLS_ASYMMETRIC_CONTENT_LEN is not set
+# CONFIG_MBEDTLS_HARDWARE_SHA is not set
```

Comment:
CONFIG_MBEDTLS_HARDWARE_SHA should be enabled (https://github.com/espressif/arduino-esp32/issues/976)